src/drone/cli.rs

```diff
@@ -46,6 +46,18 @@ pub struct Opts {
     #[clap(long, action)]
     pub acme_server: Option<String>,
 
+    //email to use as mailto for cert issuance
+    #[clap(long, action)]
+    pub cert_email: Option<String>,
+
+    /// EAB ID
```

Comment:
Replace `EAB` with `ACME External Account Binding` for clarity, here and also below.

Might also be good to explain that this is only used for ACME providers other than Let's Encrypt.

Reply:
kinda ironic, considering the IETF came up with it for them :laughing: . Noted.